test/test.hs

```diff
@@ -736,13 +741,8 @@ tests = testGroup "hevm"
           s = Expr.simplify e
         equal <- checkEquiv e s
         assertEqualM "Must be equal" True equal
-    , test "copy-slice-overflowing-into-too-short-buffer" $ do
```

Comment:
This test was basically a duplicate of copy-slice-overflow-beyond-source-into-nonempty